main.cpp

```diff
@@ -52,84 +52,86 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: cat
+//  action 1: the cat meows
+cat.meows();
+//  action 2:the cat cleans itself
+cat.cleanBody();
+//  action 3: the cat purrs
+cat.purr();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: truck
+//  action 1: The truck make sound through the horn
+truck.honkHorn();
+//  action 2: the truck applies engine power to the wheels
+truck.applyPowerToWheels();
+//  action 3:the truck changes the gear ration in the automatic transimssion
+truck.changeGear();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:Basketball Player
+//  action 1: the player shoots a jump shot
+bballPlayer.shootsJumper();
+//  action 2:The player attempts to block a shot
+bballPlayer.blockShot();
+//  action 3:  The player celebrates
+bballPlayer.celebrates();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: Baby
+//  action 1: The baby eats
+baby.eat();
+//  action 2: The baby sleeps
+baby.sleep();
+//  action 3: the baby poops
+baby.poop();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: Cell phone
+//  action 1: Cell phone makes call
+cellPhone.call();
+//  action 2: Cell phone display text message
+cellPhone.displayTextMessage();
+//  action 3: Cell phone unlocks screen
+cellPhone.unlockScreen();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: Keyboard
+//  action 1: Keyboard transmits character
+keyboard.transmitCharacter();
+//  action 2: Keyboard toggles Caps Lock indicator
+keyboard.toggleCapsLockLED();
+//  action 3: Keyboard connects to PC
+keyboard.connectToPC();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  Noun:Vacuum Cleanner
+//  action 1: Vacuum cleaner engages suction motor
+vacuumCleaner.engageMotor();
+//  action 2: Vacuum Cleaner releases hose
+vacuumCleaner.releaseHose();
+//  action 3: Vacuum cleaner engages spining brush
+vacuumCleaner.engageBrush();
 
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: Amplifier
+//  action 1: Amplifier enables output
+amplifier.enableOutput();
+//  action 2: amplifier modifies volume
+amplifier.modifyVolume();
+//  action 3: amplifier adjusts treble control gain
+amplifier.modifyTrebleGain();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.